deployment/hasura/migrations/AerieScheduler/0_init/tables/schema_migrations.sql

```diff
@@ -0,0 +1,23 @@
+create table "schema_migrations" (
+  "version" varchar not null primary key
+);
+
+create function mark_migration_applied(migration_version varchar)
+returns void
+language plpgsql as $$
```

Comment:
```suggestion
create procedure mark_migration_applied(migration_version varchar)
language plpgsql as $$
```

And similarly for `mark_migration_rolled_back`/other void functions.

Reply:
From what I understand, the primary purpose of procedures, which differentiates them from functions, is the ability to begin and commit transactions. If we don't need that power, I recommend that we use void functions instead.

Am I perhaps overlooking a reason not to use void functions?

Reply:
From my understanding, while procedures *can* begin/end transactions, [based on how the documentation is written](https://www.postgresql.org/docs/14/xproc.html), it seems like that's just an additional thing it can do, while its main distinction is that, unlike a function, it doesn't have a return value. 

It's also worth noting that we currently have no void functions in our db (unless it didn't appear in search, in which case, my bad!), so I guess my response to "is there a reason not to use void functions" is "is there a reason *to* use void functions here?" If so, then I don't mind these staying as void functions.